git-quick-stats

```diff
@@ -141,6 +167,93 @@ function commitsCalendarByAuthor() {
     '
 }
 
+# DESC: Shows a heatmap of commits per hour of each day for the last 30 days
+function commitsHeatmap() {
+    optionPicked "Commit Heatmap for the last $_commit_days days"
+
+    color_for_count() {
+        local n=$1
+        if (( n == 0 )); then
```

Comment:
Looking at this again. Should this be `n == 1`? Asking because 0 commits is technically the grey dot. I think the overall output is the same regardless, though. Up to you if you want to adjust this or not.